src/cli.rs

```diff
@@ -527,6 +527,35 @@ pub fn build_cli() -> Command {
                         .help("Path to a Docker config.json-like path. Can be used to indicate registry authentication details")
                 )
         )
+        .subcommand(
+            Command::new("save")
+                .about("save policies in tar file")
+                .arg(
+                    Arg::new("policies")
+                        .num_args(1..)
+                        .required(true)
+                        .help("list of policies to save")
+                )
+                .arg(
+                    Arg::new("output")
+                    .long("output")
+                    .short('o')
+                    .required(true)
+                    .value_name("PATH")
+                    .help("path where the file will be stored")
+                )
+
+        )
+        .subcommand(
+            Command::new("load")
+                .about("load policies from tar file")
+                .arg(
+                    Arg::new("input")
+                        .long("input")
+                        .required(true)
```

Comment:
Also in this case, by being required, it's not possible to read the contents of the policy straight from STDIN